packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.test.tsx

```diff
@@ -205,6 +205,18 @@ describe('ComboBox', () => {
     expect(inputElement.value).toEqual('1');
   });
 
+  it('Multiselect does not mutate props', () => {
+    domNode = renderIntoDocument(<ComboBox selectedKey="1" options={DEFAULT_OPTIONS} multiSelect />);
+
+    const buttonElement = domNode.querySelector('.ms-ComboBox button')!;
+    ReactTestUtils.Simulate.click(buttonElement);
+
+    const buttons = document.querySelectorAll('.ms-ComboBox-option > input');
+    ReactTestUtils.Simulate.change(buttons[1]);
+
+    expect(!!DEFAULT_OPTIONS[1].selected).toEqual(false);

```

Comment:
Do you actually expect this to be undefined? There is a `toBeUndefined` assertion if so

Reply:
I expect it to be undefined or false. Combobox does a bad job of dealing with them, since you can pass in options that don't have selection state built in.